pages.it/common/assimp.md

```diff
@@ -0,0 +1,32 @@
+# assimp
+
+> Client da linea di comando per la Open Asset Import Library.
+> Supporta il caricamento di 40+ formati di file per modelli 3D, e l'espoerazione di diversi formati 3D popolari.
+
+- Elenca tutti i formati supportati:
+
+`assimp listext`
+
+- Elenca tutti i formati supportati per l'esportazione:
+
+`assimp listexport`
+
+- Converti un file a uno dei tipi supportati, utilizzando i parametri predefiniti:
+
+`assimp export {{file_input.stl}} {{file_output.obj}}`
```

Comment:
Sometimes you've translated the filenames, and sometimes you haven't - as here.

Reply:
That is indeed translated. "*Input file*" becomes "*file input*" in italian. 

The word "input" is used in Italian too (in this case as an adjective), the only difference is that the adjective in italian comes after the name. 

"*This is an input file*" → "*questo è un file input*" or "*questo è un file di input*".

Reply:
Ah, I see! Cool :D